notebooks/README.md

```diff
@@ -13,81 +13,80 @@ You also need to install the below packages,
+- [pdf](https://github.com/QuantumBFS/tutorials/raw/master/notebooks/pdf/2-gates_and_qubits.pdf), [Pluto Notebook](https://raw.githubusercontent.com/QuantumBFS/tutorials/master/notebooks/Pluto/2-gates_and_qubits.jl)
 
-[3 - Using Yao](https://htmlpreview.github.io/?https://github.com/QuantumBFS/tutorials/blob/master/Notebooks/html/3-yao_code.html)
+[3 - Using Yao](https://htmlpreview.github.io/?https://github.com/QuantumBFS/tutorials/blob/master/notebooks/html/3-yao_code.html)
 - Basics of building a circuit in Julia using Yao- 
-- [pdf](https://github.com/QuantumBFS/tutorials/raw/master/Notebooks/pdf/3-yao_code.pdf), [Pluto Notebook](https://raw.githubusercontent.com/QuantumBFS/tutorials/master/Notebooks/Pluto/3-yao_code.jl)
+- [pdf](https://github.com/QuantumBFS/tutorials/raw/master/notebooks/pdf/3-yao_code.pdf), [Pluto Notebook](https://raw.githubusercontent.com/QuantumBFS/tutorials/master/notebooks/Pluto/3-yao_code.jl)
 
-[4 - Applications](https://htmlpreview.github.io/?https://github.com/QuantumBFS/tutorials/blob/master/Notebooks/html/4-applications.html)
+[4 - Applications](https://htmlpreview.github.io/?https://github.com/QuantumBFS/tutorials/blob/master/notebooks/html/4-applications.html)
 - Bell Circuit, Reverse Bell Circuit and Bell State
 - Superdense Coding
 - Quantum Teleportation
 - References - [Quantum Computing for Everyone](https://mitpress.mit.edu/books/quantum-computing-everyone)
-- [pdf](https://github.com/QuantumBFS/tutorials/raw/master/Notebooks/pdf/4-applications.pdf), [Pluto Notebook](https://raw.githubusercontent.com/QuantumBFS/tutorials/master/Notebooks/Pluto/4-applications.jl)
+- [pdf](https://github.com/QuantumBFS/tutorials/raw/master/notebooks/pdf/4-applications.pdf), [Pluto Notebook](https://raw.githubusercontent.com/QuantumBFS/tutorials/master/notebooks/Pluto/4-applications.jl)
 
-[Assignment 1](https://raw.githubusercontent.com/QuantumBFS/tutorials/master/Notebooks/Pluto/assignment1.jl)
+[Assignment 1](https://raw.githubusercontent.com/QuantumBFS/tutorials/master/notebooks/Pluto/assignment1.jl)
 
 #### Intermediate
 
-[5 - More Quantum Gates](https://htmlpreview.github.io/?https://github.com/QuantumBFS/tutorials/blob/master/Notebooks/html/5-more_gates.html)
+[5 - More Quantum Gates](https://htmlpreview.github.io/?https://github.com/QuantumBFS/tutorials/blob/master/notebooks/html/5-more_gates.html)
 - More Single Qubit gates
 - More Multi Qubit gates
 - Some Blocks
 - References - [Yao Documentation](https://docs.yaoquantum.org/)
-- [pdf](https://github.com/QuantumBFS/tutorials/raw/master/Notebooks/pdf/5-more_gates.pdf), [Pluto Notebook](https://raw.githubusercontent.com/QuantumBFS/tutorials/master/Notebooks/Pluto/5-more_gates.jl)
+- [pdf](https://github.com/QuantumBFS/tutorials/raw/master/notebooks/pdf/5-more_gates.pdf), [Pluto Notebook](https://raw.githubusercontent.com/QuantumBFS/tutorials/master/notebooks/Pluto/5-more_gates.jl)
```

Comment:
ah I didn't realize you were using an absolute link, you can just use `pdf/5-more_gates.pdf` etc, but note PDF is not included in the repo, I think we should remote this link.

Reply:
The pdf part is there for when the Github Actions part is complete

Reply:
yeah but the link is wrong, it will be in the release CDN. let's remote this link for now.